mods/ra/maps/fort-lonestar/map.yaml

```diff
@@ -563,16 +577,31 @@ Rules:
 		CashTrickler:
 			Period: 250
 			Amount: 50
+	MOBILETENT:
+		Inherits: MCV
+		Transforms:
+			IntoActor: tent
+			Offset: 0,0
+		RenderSprites:
+			Image: MCV
+			Scale: 0.5
```

Comment:
That's irrelevant. This is just bad coding based on a legacy option.

The new `WithDecoration` proposal denies the use of that trait for WW-styled `EMP`/mindcontrol/`AttachEffect.Animation` usecases - forcing `WithIdleOverlay` for such which gets affected by `Scale` and ultimately leads into severe issues due to some passive encouragement from usecases like here.